src/pages/dictionaries/DictionariesIndex.tsx

```diff
@@ -89,14 +89,13 @@ const getExternalLink = (
   );
 
 const DictionariesIndex = ({
-  dictionaryIndex,
-  setDictionaryIndex,
   fetchAndSetupGlobalDict,
   globalLookupDictionary,
   globalLookupDictionaryLoaded,
   stenohintsonthefly,
   userSettings,
 }: Props) => {
+  const dictionaryIndex = useAtomValue(dictionaryIndexState);
```

Comment:
`dictionaryIndexState` is a lazy atom, and before loading, it will suspend and nearest surrounding `<Suspense>` boundary would catch and render loading state.